packages/sui-ssr/server/criticalCss/index.js

```diff
@@ -115,8 +123,34 @@ export default config => (req, res, next) => {
           })
 
           res.on('end', () => {
-            logMessage(`Add cache entry for ${hash}`)
             __REQUESTING__ = false
+
+            // Check if any currentConfig mandatory CSS rule is missing in generated critical CSS
+            if (
```

Comment:
Let's refactor this conditional in order to be readable and mantainable by humans. :D